sass/directives/03_components/_card_group.scss

```diff
@@ -0,0 +1,159 @@
+@mixin card-group {
+  .card-group__items {
+    @include card-deck--centered;
+
+    margin: -($base-spacing-medium / 2);
+  }
+
+  .card-group__card {
+    @include card--tile($include_strip: false);
+    // Restore the original box-shadow since it is lost when adding layers. This is not the same as $base-box-shadow-hover
+    box-shadow: -1px 2px 1px 0px rgba(0, 0, 0, 0.25);
```

Comment:
there should be a `base-box-shadow` variable or something like that, in sizing i think

Reply:
I'm going to make this into a variable actually. The reason for this is that due to layering of these cards, base-box-shadow and base-box-shadow-hover are not the same with these cards compared to our normal cards. Using base-box-shadow with these cards actually produces a shadow that really cannot be seen at all.

Reply:
👍